### DISCUSSION TITLE
What all feature you want from `ksctl`

### DISCUSSION THREAD

**Discussion by dipankardas011** [kubesimplify/ksctl#208] [Ideas]
Love to see your comments 🤗

**Comment by aj11anuj** [comment]
As there will be many people who will get to know about ksctl, when they have got familiar with command-line of different cloud providers or tool. It will be better, if ksctl will be able to provide people that familiar experience and not the one like they are learning another new CLI tool.

**Reply by dipankardas011** [reply]
Sure that's why we are planning for a UI dashboard for anyone  who is not that comfortable with CLI
